draft-ietf-httpbis-client-hints.md

```diff
@@ -56,11 +56,19 @@ informative:
       ins: Y. Weiss
       name: Yoav Weiss
       organization: Google
+  CLIENT-HINTS-INFRASTRUCTURE:
+    target: https://wicg.github.io/client-hints-infrastructure/
+    title: Client Hints Infrastructure
+    author:
+    -
+      ins: Y. Weiss
+      name: Yoav Weiss
+      organization: Google
 
 
 --- abstract
 
-HTTP defines proactive content negotiation to allow servers to select the appropriate response for a given request, based upon the user agent's characteristics, as expressed in request headers. In practice, clients are often unwilling to send those request headers, because it is not clear whether they will be used, and sending them impacts both performance and privacy.
+HTTP defines proactive content negotiation to allow servers to select the appropriate response for a given request, based upon the client's characteristics, as expressed in request headers. In practice, clients are often unwilling to send those request headers, because it is not clear whether they will be used, and sending them impacts both performance and privacy.
```

Comment:
Here and elsewhere, changing "user agent" to "client" implies that the scope of discussion -- including requirements -- includes intermediaries as well as UAs. Is that intentional?

Reply:
That's a good point, I haven't considered intermediaries. One of the points raised in the review was to align on either "client" or "user agent". I guess it'd be more appropriate to align on the latter.

Reply:
Aligned on user agent. Let me know if that works